### ISSUE TITLE
Gradle build Failed"Task :app:mergeDebug Resources FAILED"  Aapt2InternalException:AAPT2 aapt2-3.4.0-5326820-windows Daemon #0: Daemon startup failed

### ISSUE THREAD

**Issue by EdifierX** [gradle/gradle#9376]
### Issue:
No matter Jave or Kotlin project  build failed
![{SBFS3P )J0_QGZIP3DMK9B](https://user-images.githubusercontent.com/26271167/57498702-8c79b180-730f-11e9-86db-8714dd132291.png)
![3FM33W)A1(F4QI( FJVPU5B](https://user-images.githubusercontent.com/26271167/57498722-a2877200-730f-11e9-96f3-fd48e229b9d1.png)

### 

### 

### 

### My Environment
![LSY5DFIA7W1](https://user-images.githubusercontent.com/26271167/57498909-50931c00-7310-11e9-9e34-84ef7082214b.png)
Android Studio Version:3.4
Window 7 
java jdk 1.8.0_101

**Comment by FireLizard** [comment]
Same probleme here :(

My Environment:
- Alpine Linux 3.9 (Kernel: 5.4.14-100.fc30.x86_64)
- openjdk 1.8.0_242
- Gradle 4.10.3
(works on Debian Stretch)

**Comment by FireLizard** [comment]
The problem is that [Alpine Linux ships with Musl instead of GNU libc, the Android SDK tooling requires glibc.](https://stackoverflow.com/a/49936412).

**Comment by cfJand** [comment]
wo have same issue, anyone help?

**Comment by bombasticbob** [comment]
I have been having this problem using gradle 4.1.2 and 4.1.3 .
I tried gradle 4.0.0 [randomly trying versions] and IT WORKED PROPERLY!!!

This stack overflow describes a similar problem that was fixed by using 4.0.2
https://stackoverflow.com/questions/65190931/android-studio-with-aapt2-windows-problems/67078572#67078572

Apparently version 4.1.x of gradle introduced this problem.
Here is the uname for the Linux (Devuan) I'm using

Linux xxxx 4.19.0-12-amd64 #1 SMP Debian 4.19.152-1 (2020-10-18) x86_64 GNU/Linux

Again, my solution is to DOWNgrade the gradle plugin version to 4.0.0 or 4.0.2 (I just tested 4.0.2)

AAPT2 fails on files over "a certain size".  When I remove icon resources larger than 'xhdpi' then the build fails stop, but only because the files causing the bug to surface are no longer there.  So whatever was done to AAPT2 since version 4.0.2 is the problem.

Simple fix:  put it BACK the way it was!
How can I accelerate the handling of this bug?  It has been documented for MONTHS, and has NOT been resolved, even with a version of Gradle that Android Studio recently released...

Is the new AAPT2 even WORKING for anyone else?
